doc/releases/index.rst

```diff
@@ -58,8 +59,11 @@ receive bug fixes or backports).
 Release timeline
 ----------------
 
-.. ceph_timeline_gantt:: releases.yml quincy pacific
-.. ceph_timeline:: releases.yml quincy pacific
+.. ceph_timeline_gantt:: releases.yml reef quincy 
+.. ceph_timeline:: releases.yml reef quincy 
```

Comment:
Pacific is not EOL yet -- there is going to be at least one more release.  Should it be retained in the timeline for now?

Related: we are still listing 2023-06-01 as the EOL date for pacific and this caused some queries on the mailing list.  I would change it to 2023-10-01 in this PR so follow the previous "roughly two months after a new major release is cut" practice.

Reply:
I have changed the EOL date for Pacific to "2023-10-01".

Reply:
Looks like the "should it be retained in the timeline for now" bit fell through the cracks.  As it is, folks looking at https://docs.ceph.com/en/latest/releases/#release-timeline wouldn't see pacific in the chart or in the table even though it's still supported for the next 2-3 months.